Sources/SwiftDocC/Utility/ValidatedURL.swift

```diff
@@ -181,14 +181,14 @@ private extension StringProtocol {
             for (index, character) in unicodeScalars.indexed() where !allowedCharacters.contains(character) {
                 if character == "%" {
                     // % isn't allowed in a URL fragment but it is also the escape character for percent encoding.
-                    let firstFollowingIndex  = unicodeScalars.index(after: index)
-                    let secondFollowingIndex = unicodeScalars.index(after: firstFollowingIndex)
-                    
-                    guard secondFollowingIndex < unicodeScalars.endIndex else {
+                    guard unicodeScalars.distance(from: index, to: unicodeScalars.endIndex) <= 2 else {
```

Comment:
I feel like the sign on this is wrong? we want the distance to be *at least* 2 scalars from the end, whereas it just seems like this is checking for it to be *at most* 2 scalars.

```suggestion
                    guard unicodeScalars.distance(from: index, to: unicodeScalars.endIndex) >= 2 else {
```